docs/index.html

```diff
@@ -47,6 +47,12 @@
       disabled
     />
 
+    <!-- Prism Theme -->
+    <!-- <link
+      rel="stylesheet"
+      href="//cdn.jsdelivr.net/npm/prismjs@1.29.0/themes/prism-twilight.min.css"
+    /> -->
+
```

Comment:
What does adding it here do?

That's the only problem, the rest is fine for me. Good job!

Reply:
This is a placeholder for quickly testing Prism.js theme functionality. This is a new feature that was added in the v5 style update. Our `index.html` contains similarly commented resources like different Vue.js versions.